### ISSUE TITLE
Feature: Add OS in debug info

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1123]
Will be useful when fixing issues. Many people create issues and for some reason add the OS of the mobile device which is irrelevant.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.1.1](https://github.com/open-wa/wa-automate-nodejs/compare/3.1.0...3.1.1) (2020-11-30)

- build(deps): bump ts-json-schema-generator from 0.78.0 to 0.79.0 [`#1110`](https://github.com/open-wa/wa-automate-nodejs/pull/1110)
- build(deps): bump swagger-ui-dist from 3.37.0 to 3.37.2 [`#1111`](https://github.com/open-wa/wa-automate-nodejs/pull/1111)
- build(deps-dev): bump release-it from 14.2.1 to 14.2.2 [`#1112`](https://github.com/open-wa/wa-automate-nodejs/pull/1112)
- build(deps-dev): bump typedoc-plugin-pages from 1.0.1 to 1.1.0 [`#1119`](https://github.com/open-wa/wa-automate-nodejs/pull/1119)
- build(deps-dev): bump prettier from 2.2.0 to 2.2.1 [`#1120`](https://github.com/open-wa/wa-automate-nodejs/pull/1120)
- build(deps-dev): bump @types/node from 14.14.9 to 14.14.10 [`#1109`](https://github.com/open-wa/wa-automate-nodejs/pull/1109)
- build(deps): bump @open-wa/wa-decrypt from 2.0.1 to 3.0.0 [`#1106`](https://github.com/open-wa/wa-automate-nodejs/pull/1106)
- Release 3.1.1 [`7ebe598`](https://github.com/open-wa/wa-automate-nodejs/commit/7ebe598402fe7419db7d21b6d6a6e2ec4a538948)
- fix cname in documentation [`6e9dabd`](https://github.com/open-wa/wa-automate-nodejs/commit/6e9dabd3f6237a024cc28417a9bb7d2e1a72deac)
- fixed `popup` not working on Windows and Linux #1122 [`c240499`](https://github.com/open-wa/wa-automate-nodejs/commit/c24049985c6cc887c18af399103bf727c6741af1)
- Added OS info to debug/session info object #1123 [`eeb1792`](https://github.com/open-wa/wa-automate-nodejs/commit/eeb1792748dfecda90466d68684cd4837659084c)
- patched: #1118 [`f761e56`](https://github.com/open-wa/wa-automate-nodejs/commit/f761e56a5a88b238c7ae6fb10bf687a7da47755d)
- fix qr launch errors on windows #1121 [`88f1822`](https://github.com/open-wa/wa-automate-nodejs/commit/88f182247ccaec394d63eb0036b51eea4631288d)
- center and crop profile picture perfectly for every aspect ratio #1105 [`2ed4770`](https://github.com/open-wa/wa-automate-nodejs/commit/2ed47706e35344e0041060e695a06b92ac4f5df8)
- patch: fail with error string instead of exception #1082 [`36e8e95`](https://github.com/open-wa/wa-automate-nodejs/commit/36e8e951492d392297907d4debde999be3c3d791)
- patch: #1105 [`e1a18ce`](https://github.com/open-wa/wa-automate-nodejs/commit/e1a18cefcdfd5a28c5ae55363b3dd995ef2ae3ff)
- Create CNAME [`45e6d60`](https://github.com/open-wa/wa-automate-nodejs/commit/45e6d60507f596f2d420796d66625e66af77c1ca)
